utils/qfirehose/Makefile

```
@@ -0,0 +1,57 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=qfirehose
+PKG_VERSION:=1.4.9
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/nippynetworks/qfirehose/archive/refs/tags/
```

```suggestion
PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
PKG_SOURCE_URL:=https://codeload.github.com/nippynetworks/qfirehose/tar.gz/$(PKG_VERSION)?
```

`$(PKG_VERSION).tar.gz` is really unrecognizable and it can cause conflict.

---

I agree. I wasn't aware how codeload works, this is great- I hate it when packages come in vX.X archives. Never can know what it is and it is possible that filenames collide.

Fixed with [03530f8](https://github.com/openwrt/packages/commit/03530f8c67b52bbebebf11299d4c414081f6989d)